docs/layouts/partials/header/header.html

```diff
@@ -0,0 +1,223 @@
+{{ if .Site.Params.alert -}}
+  {{ partial "header/alert.html" . }}
+{{ end -}}
+
+{{ if eq .Site.Params.options.navbarSticky true -}}
+<div class="sticky-top">
+{{ end -}}
+
+<div class="header-bar"></div>
+
+<header class="navbar navbar-expand-lg navbar-light doks-navbar">
+  <nav class="container-{{ if .Site.Params.options.fullWidth }}fluid{{ else }}xxl{{ end }} flex-wrap flex-lg-nowrap" aria-label="Main navigation">
+
+    <a class="navbar-brand order-0" href="{{ .Site.BaseURL | relLangURL }}" aria-label="{{ .Site.Params.Title }}">
+      {{ .Site.Params.Title }}
+      <svg width="30" height="30" viewBox="0 0 152.735065 152.735065" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <title>DJ 2</title>
```

Comment:
Is this a typo?

Reply:
This is just an SVG definition, so the title doesn't actually matter.